.github/workflows/tests.yml

```diff
@@ -67,7 +67,7 @@ jobs:
       run: sudo apt-get update
       # gpgme-devel is needed for container upload dependencies
     - name: Install test dependencies
-      run: sudo apt-get install -y libgpgme-dev
+      run: sudo apt-get install -y libgpgme-dev libbtrfs-dev
```

Comment:
Seems like an unrelated change? Could we split this out to a separate commit?

Reply:
can be even part of #4431

Reply:
The DB tests seem to fail if this is missing :-/
I'll add it for now (although I don't understand why this is needed now)